### PR TITLE
Updated the intro and added a few expectations

### PR DESCRIPTION
**Pull request by jeffmaher**
Updated the text to set expectations for what someone might build with this and what some of the technology choices/defaults are.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 **Demo:** https://cds-node-starter.herokuapp.com
 
-Should you use this? It depends ;)
+This repository provides a codebase that can be used to quickly start an application for building web-based forms that look like a Government of Canada website. It's setup with some sensible defaults and tech choices, such as:
 
-Don't like the way it's setup -> it's an Express server so do your thing `app.js`
+- Node.js 10.x
+- [Express](https://expressjs.com/) web framework
+- [Pug](https://pugjs.org/) view templates
```
Comment (jeffmaher) [inline README.md]: Are there any other defaults/choices that would be good to list?

Comment (timarney) [reply README.md]: Should add SASS

Comment (maxneuvians) [reply README.md]: Should we push people to use NVM?

Comment (timarney) [reply README.md]: @maxneuvians yes.

Comment (jeffmaher) [reply README.md]: Cool. Will add these in.

Comment (jeffmaher) [reply README.md]: Added NVM and Sass. Resolving.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
-# Node Starter App
-
-Quick start application setup with sensible defaults baked in.
+# Canada Web Forms Starter Repo
 
 **Demo:** https://cds-node-starter.herokuapp.com
 
-Should you use this? It depends ;)
+This repository provides a codebase that can be used to quickly start an application for building web-based forms that look like a Government of Canada website. It's setup with some sensible defaults and tech choices, such as:
```
Comment (timarney) [inline README.md]: Note: It could just be a page with a few views and no forms so might want to reword this.

Comment (jeffmaher) [reply README.md]: Good suggestion! Rephrased. How does it look now?

**Pull request merged**